### PR TITLE
Initial draft of how to create and use mysql stored procedures

### PR DESCRIPTION
**Pull request by francisndungu**
Greetings,

This is my initial guide on how to create and use MySQL database triggers.

Hoping for the best.

Best wishes,

Francis Ndungu

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 59853a092c8161c26bd27b240418741437e8bf9b

https://deploy-preview-2903--nostalgic-ptolemy-b01ab8.netlify.com

File: docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+1.  Next, you will see a MySQL prompt similar to the one shown below.
+
+    {{< output >}}
+mysql >
+{{< /output >}}
+
+1.  Enter the command below to create a `test_db` database:
+
+        Create database test_db;
```
Comment (andystevensname) [inline docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md]: We should try to stick to a convention for commands, either we put them in capitals or keep them all lowercase.  A mix are used in this guide.

File: docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+        Insert into products (product_id, product_name, category_name) values ('1', 'GAMING KEYBOARD', 'COMPUTER ACCESSORIES');
+        Insert into products (product_id, product_name, category_name) values ('2', 'OPTICAL MOUSE', 'COMPUTER ACCESSORIES');
+        Insert into products (product_id, product_name, category_name) values ('3', 'MOUSE PAD', 'COMPUTER ACCESSORIES');
+        Insert into products (product_id, product_name, category_name) values ('4', 'STEREO SYSTEM', 'ELECTRONICS');
+        Insert into products (product_id, product_name, category_name) values ('5', '32 INCH TV', 'ELECTRONICS');
+        Insert into products (product_id, product_name, category_name) values ('6', 'DVB-T2 RECEIVER', 'ELECTRONICS');
+
+    You should get the below output after executing each `Insert` statement.
+Select * from products where category_name=category;
```
Comment (andystevensname) [inline docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md]: This should be indented to make a code block

File: docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+
+    - The parameter can be changed during stored procedure execution.
+
+    - The changed value can be returned back to the calling program, if the calling program passed a variable as the parameter.
+
+### An Example Stored Procedure
+
+After understanding the basic syntax, let's create a simple stored procedure to filter products by category name. The category name will be supplied as an `IN` parameter.
+
+1. Log in to the MySQL server using the test_user's credentials that you created when preparing the database:
```
Comment (andystevensname) [inline docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md]: Should test_user be in backticks here?

File: docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+3 rows in set (0.00 sec)
+
+Query OK, 0 rows affected (0.01 sec)
+{{< /output >}}
+
+Our stored procedure worked as we expected. Next, we will learn how to drop the stored procedures if we no longer want them to execute again.
+
+## Deleting Stored Procedures
+
+You can delete a MySQL stored procedure if you no longer want to use it or if you want to recreate it from scratch. The basic syntax of dropping the stored procedure is shown below
```
Comment (andystevensname) [inline docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md]: Need a semicolon after "shown below"

File: docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md (hunk 3 above)
Comment (nmelehan) [reply docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md]: ✅

File: docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md (hunk 2 above)
Comment (nmelehan) [reply docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md]: This code was actually an accidental copy-paste that didn't need to be there, so I removed it

File: docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md (hunk 1 above)
Comment (nmelehan) [reply docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md]: Agreed; changed to all caps across guide, with the exception of user-defined stored procedures

File: docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md (hunk 4 above)
Comment (nmelehan) [reply docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md]: ✅

**Pull request merged**